src/common.cpp

```diff
@@ -444,6 +444,7 @@ void shim::add_malloc_shimmed_symbols(std::vector<shim::shimmed_symbol> &list) {
         {"posix_memalign", ::posix_memalign},
         {"_Znwj", (void *(*)(size_t)) ::operator new},
         {"_ZdlPv", (void (*)(void *)) ::operator delete},
+        {"mallinfo", ::mallinfo},
```

Comment:
Sorry I can't agree merging this, since this doesn't compile on macOS.

Reply:
Fair, right now though I'm just trying to explore what it would take to get it working at all.  These PRs are for reference as part of a larger wip.

So far the furthest I've gotten is a black window that eventually crashes and spams about the Game context being gained.  I'll post a more detailed write up tomorrow.